floris/simulation/farm.py

```diff
@@ -25,6 +25,7 @@
     State,
     Turbine,
 )
+from floris.simulation.turbine import _compute_tilt_angles_for_floating_turbines
```

Comment:
Since `_compute_tilt_angles_for_floating_turbines` is imported outside of turbine, it should not have a prefixed underscore, or there's a reason it is meant to be private it should not be imported directly here.

Reply:
Updated to be a public method.

Reply:
I think you may have missed that change in your commit, I still see it with the underscore in the pull request

Reply:
Had it locally, didn't push it up. Should be there now.